source/deployment/sso-saml-okta.rst

```diff
@@ -44,30 +42,38 @@ If you’d like to use a Mattermost logo for the application, you can download o
 
 	.. image:: ../../source/images/okta_5_advanced_saml_settings.PNG
 
-9. Enter attribute statements, which will be used to map attributes between Okta and Mattermost. For more information on which attributes are configurable, see our :ref:`documentation on SAML configuration settings <saml-enterprise>`. Email and username attributes are required. For Mattermost servers running 3.3 and earlier, first name and last name attributes are also required.
+9. Enter attribute statements used to map attributes between Okta and Mattermost. For more information on which attributes are configurable, see our `documentation on SAML configuration settings <https://docs.mattermost.com/administration/config-settings.html#saml>`__. Email and username attributes are required. For SAML with Okta, an `ID attribute <https://docs.mattermost.com/administration/config-settings.html#id17>`__ is also required, and that ID must be mapped to ``user.id``. For Mattermost servers running 3.3 and earlier, first name and last name attributes are also required.
```

Comment:
How many servers are still using 3.3? Maybe we can take this line out? :)

Reply:
Given that v3.3 servers aren't represented in our [Looker-based Server & Client Version Usage data](https://mattermost.looker.com/dashboards/172?Mobile%20Client%20Usage=&filter_config=%7B%22Mobile%20Client%20Usage%22:%5B%7B%22type%22:%22any%22,%22values%22:%5B%7B%22constant%22:%22%22%7D,%7B%7D%5D,%22id%22:1%7D%5D%7D), and that our data only starts at v4.0, under most circumstances, I feel we'd be safe to remove this information. However, I'd like to complete some due diligence first to ensure we won't be negatively impacting customers relying on these details by removing them.

Reply:
I'm comfortable with removing 3.3. It's way, way past supported so the first they should do is upgrade

Reply:
Thanks, @icelander! I'll remove the references to v3.3 here.